### DISCUSSION TITLE
Midnight till now?

### DISCUSSION THREAD

**Discussion by andygodber** [RomRider/apexcharts-card#755] [Q&A]
I came here by a couple of routes as this looked to be a great solution for many needs!
My immediate one is how to graph from midnight until "now"? Ive seen several posts, including @RomRider own at https://github.com/kalkih/mini-graph-card/issues/486#issuecomment-770458689 saying its possible, but I can only get a chart that shows full 24 hours.
The problem is that any time beyond 'now' is shown as the last value.
I feel its an obvious answer, but I couldnt spot it ;)
TFYH.

**Comment by costa-alex** [comment]
Same question! I always get a line with the last known value until the end of day.

**Comment by sjengfred** [comment]
You can add the `extend_to: now` option for the series. 

See also: https://github.com/RomRider/apexcharts-card?tab=readme-ov-file#series-options

**Reply by costa-alex** [reply]
Thanks, missed that option when configuring the chart.
